gateway/1.5/modules/ROOT/pages/flex-local-publish-api-multiple-services.adoc

```diff
@@ -92,13 +113,14 @@ The response looks something like this:
 
 An API with multiple upstream services is now running behind Flex Gateway.
 
+[[docker]]
 == Publish an API Running Behind Flex Gateway in a Docker Container 
 
 === Before You Begin
 
-* Flex Gateway downloaded. See xref:flex-install.adoc[] for more information. 
+* Flex Gateway downloaded. For more information, see xref:flex-install.adoc[]. 
```

Comment:
These Before You Begin sections should be consistent in the doc:

Ensure that you have: